website/docs/ide.md

```diff
@@ -54,3 +56,30 @@ The following Metals features are expected to work, among others:
 
 Upon adding new dependencies, via `scala.conf` or via `import $dep` in Scala sources, the
 new dependencies should be automatically downloaded and be available right after in Metals.
+
+## Neovim
+
+### Setup
+
+You can get Scala CLI support in Neovim by using
+[`nvim-metals`](https://github.com/scalameta/nvim-metals). Currently Scala CLI
+support relies on a custom Metals server for now. To enable it make sure to set
```

Comment:
NIT
```suggestion
[`nvim-metals`](https://github.com/scalameta/nvim-metals). Scala CLI
support relies on a custom Metals server for now. To enable it make sure to set
```
(keep either "currently" or "for now"? - I think there was the same thing in earlier Scala CLI docs, maybe you copied it from there?)

Reply:
Ahhh, yea that's my mistake, I copied and then didn't fully read 😬 I'll fix this.